src/common/catalog-entities/kubernetes-cluster.ts

```diff
@@ -19,6 +20,38 @@ export class KubernetesCluster extends CatalogEntity<CatalogEntityMetadata, Kube
   public readonly apiVersion = "entity.k8slens.dev/v1alpha1";
   public readonly kind = "KubernetesCluster";
 
+  async connect(): Promise<boolean> {
```

Comment:
I don't think these need to return anything. Especially since it is currently not very useful of a return value.

Reply:
Changed to `Promise<void>`.